flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/java/windows.scala

```diff
@@ -98,7 +98,7 @@ object Over {
     */
   def orderBy(orderBy: String): OverWindowWithOrderBy = {
     val orderByExpr = ExpressionParser.parseExpression(orderBy)
-    new OverWindowWithOrderBy(Seq[Expression](), orderByExpr)
+    new OverWindowWithOrderBy(Array[Expression](), orderByExpr)
```

Comment:
Good catch!
And suggest to change the `java/windows.scala` as well. What do you think?